### ISSUE TITLE
Tilde in config breaks dhall

### ISSUE THREAD

**Issue by domenkozar** [cachix/cachix#380]
```
╰─❯ cachix use yyy

cachix:

Error: Invalid input


(input):1:50:

|

1 | /Users/xxx/Library/Mobile Documents/com~apple~CloudDocs/System/mackup/.config/cachix/cachix.dhall

| ^

unexpected '~'

expecting "@", ->, :, end of input, operator, or whitespace
```

**Comment by shepting** [comment]
That path is the full path on the system, but Cachix is initially looking at `/Users/xxx/.config/cachix` which is a symlink to that longer path.

Thanks for having a look, I appreciate it.

**Comment by domenkozar** [comment]
I wasn't able to reproduce, but playing around with primitives I found https://github.com/dhall-lang/dhall-lang/issues/1188

**Comment by domenkozar** [comment]
@shepting I've released Cachix 0.6.1 that hopefully fixes this!
